swift/Samples/Tutorial/Frameworks/Tutorial5Complete/Sources/Todo/List/TodoListWorkflow.swift

```diff
@@ -35,100 +33,81 @@ struct TodoListWorkflow: Workflow {
     }
 }
 
-
 // MARK: State and Initialization
 
 extension TodoListWorkflow {
-
-    struct State {
-    }
+    struct State {}
 
     func makeInitialState() -> TodoListWorkflow.State {
         return State()
     }
 
-    func workflowDidChange(from previousWorkflow: TodoListWorkflow, state: inout State) {
-
-    }
+    func workflowDidChange(from _: TodoListWorkflow, state _: inout State) {}
 }
 
-
 // MARK: Actions
 
 extension TodoListWorkflow {
-
     enum Action: WorkflowAction {
-
         typealias WorkflowType = TodoListWorkflow
 
         case onBack
         case selectTodo(index: Int)
         case new
 
-        func apply(toState state: inout TodoListWorkflow.State) -> TodoListWorkflow.Output? {
-
+        func apply(toState _: inout TodoListWorkflow.State) -> TodoListWorkflow.Output? {
             switch self {
-
             case .onBack:
                 // When a `.onBack` action is received, emit a `.back` output
                 return .back
 
-            case .selectTodo(index: let index):
+            case let .selectTodo(index: index):
                 // Tell our parent that a todo item was selected.
                 return .selectTodo(index: index)
 
             case .new:
                 // Tell our parent a new todo item should be created.
                 return .newTodo
             }
-
         }
     }
 }
 
-
 // MARK: Workers
 
 extension TodoListWorkflow {
-
     struct TodoListWorker: Worker {
-
-        enum Output {
-
-        }
+        enum Output {}
 
         func run() -> SignalProducer<Output, Never> {
             fatalError()
         }
 
-        func isEquivalent(to otherWorker: TodoListWorker) -> Bool {
+        func isEquivalent(to _: TodoListWorker) -> Bool {
             return true
         }
-
     }
-
 }
 
 // MARK: Rendering
 
 extension TodoListWorkflow {
-
     typealias Rendering = BackStackScreen.Item
 
-    func render(state: TodoListWorkflow.State, context: RenderContext<TodoListWorkflow>) -> Rendering {
-
+    func render(state _: TodoListWorkflow.State, context: RenderContext<TodoListWorkflow>) -> Rendering {
         // Define a sink to be able to send the .onBack action.
         let sink = context.makeSink(of: Action.self)
 
         let titles = todos.map { (todoModel) -> String in
-            return todoModel.title
+            todoModel.title
```

Comment:
May be i'm the only one but i like mentioning the return explicitly.

Reply:
me too, but I'm pretty sure we're in the minority.

Reply:
![](https://media.giphy.com/media/ReBGGJtbXrjbQJwByP/giphy.gif)

Reply:
This whole line should be probably be written like this now:

```swift
let titles = todos.map(\.title)
```